cfme/common/provider.py

```diff
@@ -64,19 +64,26 @@ class Credential(cfme.Credential, Updateable):
              type: One of [amqp, candu, ssh, token] (optional)
              domain: Domain for default credentials (optional)
         """
+
         @property
         def form(self):
             fields = [
                 ('token_secret_55', Input('bearer_token')),
                 ('google_service_account', Input('service_account')),
             ]
+
             tab_fields = {
                 "Default": [
                     ('default_principal', Input("default_userid")),
                     ('default_secret', Input("default_password")),
                     ('default_verify_secret', Input("default_verify")),
-                    ('token_secret', Input('bearer_password')),
-                    ('token_verify_secret', Input('bearer_verify')),
+                    ('token_secret', {
+                        version.LOWEST: Input('bearer_password'),
+                        '5.6': Input('default_password')
+                    }),
+                    ('token_verify_secret', {
+                        version.LOWEST: Input('bearer_password'),
```

Comment:
Which ones? The spaces? Or 
('token_secret', Input('bearer_password')),
('token_verify_secret', Input('bearer_verify')),